### PR TITLE
Select.wind.of.change

### PR DESCRIPTION
**Pull request by lucienbertin**
https://www.youtube.com/watch?v=n4RjJKxsamQ

complete refacto of the select

### CODE REVIEW

File: packages/ng/libraries/core/src/lib/input/input.model.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { ControlValueAccessor } from '@angular/forms';
+import { ILuPickerPanel } from './picker/input-picker.model';
+import { ILuPopoverTrigger } from '../popover';
```
Comment (GuillaumeNury) [inline packages/ng/libraries/core/src/lib/input/input.model.ts]: /index ?

File: packages/ng/libraries/core/src/lib/option/picker/option-picker.component.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+	OnDestroy,
+	forwardRef,
+} from '@angular/core';
+import { LuPopoverPanelComponent, luTransformPopover } from '../../popover/index';
+import { ILuOptionItem, LuOptionItemComponent } from '../item/index';
+import { ILuOptionPickerPanel } from './option-picker.model';
+import { Subscription } from 'rxjs/Subscription';
+import { Observable } from 'rxjs/Observable';
+import { merge } from 'rxjs/observable/merge';
+import 'rxjs/add/observable/of';
```
Comment (GuillaumeNury) [inline packages/ng/libraries/core/src/lib/option/picker/option-picker.component.ts]: ce serait bien de passer à la nouvelle syntaxe, cette façon de faire va pas tarder à mourir

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://342-pr-lucca-front-LuccaSA.surge.sh

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://342-pr-lucca-front-LuccaSA.surge.sh

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://342-pr-lucca-front-LuccaSA.surge.sh

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://342-pr-lucca-front-LuccaSA.surge.sh

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://342-pr-lucca-front-LuccaSA.surge.sh

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://342-pr-lucca-front-LuccaSA.surge.sh

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://342-pr-lucca-front-LuccaSA.surge.sh

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://342-pr-lucca-front-LuccaSA.surge.sh

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://342-pr-lucca-front-LuccaSA.surge.sh

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://342-pr-lucca-front-LuccaSA.surge.sh

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://342-pr-lucca-front-LuccaSA.surge.sh

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://342-pr-lucca-front-LuccaSA.surge.sh

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://342-pr-lucca-front-LuccaSA.surge.sh

**Pull request merged**